math/gnudatalanguage/Portfile

```diff
@@ -63,9 +60,7 @@ conflicts_build             antlr
 
 cmake.out_of_source yes
 
-patchfiles              patch-src-magick_cl.cpp.diff \
-                        patch-src-hdf_fun.cpp.diff
-patchfiles-append       sincos.patch
+patchfiles              patch-src-hdf_fun.cpp.diff
```

Comment:
If some patch files are no longer required, please remember to also delete them as part of this update.

Reply:
corrected